go/cmd/vtctldclient/command/vreplication/common/utils.go

```diff
@@ -185,8 +185,8 @@ func AddCommonCreateFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&CreateOptions.TabletTypesInPreferenceOrder, "tablet-types-in-preference-order", true, "When performing source tablet selection, look for candidates in the type order as they are listed in the tablet-types flag.")
 	cmd.Flags().StringVar(&CreateOptions.OnDDL, "on-ddl", onDDLDefault, "What to do when DDL is encountered in the VReplication stream. Possible values are IGNORE, STOP, EXEC, and EXEC_IGNORE.")
 	cmd.Flags().BoolVar(&CreateOptions.DeferSecondaryKeys, "defer-secondary-keys", false, "Defer secondary index creation for a table until after it has been copied.")
-	cmd.Flags().BoolVar(&CreateOptions.AutoStart, "auto-start", true, "Start the MoveTables workflow after creating it.")
```

Comment:
Up to you (I think it's fine how you have it!): could also pass a string arg to `AddCommonFlags` and start doing stuff like:

```go
cmd.Flags().BoolVar(&CreateOptions.AutoStart, "auto-start", true, fmt.Sprintf("Start the %s workflow after creating it.", operation))
```